q2_demux/tests/test_demux.py

```diff
@@ -787,20 +787,25 @@ def test_inconsistent_sequence_length_single(self):
 
         demux_data = emp_single(bsi, barcode_map)
         with tempfile.TemporaryDirectory() as output_dir:
-            with self.assertRaisesRegex(ValueError,
-                                        'Observed sequences of length'):
-                # TODO: Remove _PlotQualView wrapper
-                summarize(output_dir, _PlotQualView(demux_data,
-                                                    paired=False), n=2)
+            # TODO: Remove _PlotQualView wrapper
```

Comment:
I'm not sure if that makes sense.  Since the subsample is random, we wouldn't know what to expect for `min_seq_len` (I mean I guess we could check that it is either 1, 2, 3, or 4, if that's what you mean).  So all we could really be sure of is that some value of `min_seq_len` is set, and that `seqCount` and `n` are both set, but I think those are both tested a lot in other places ... what do you think?  Is it worthwhile?  Do I misunderstand the sort of test you're envisioning here?